packages/transactions/src/utils.ts

```diff
@@ -66,7 +68,8 @@ export function omit<T, K extends keyof any>(obj: T, prop: K): Omit<T, K> {
   return clone;
 }
 
-export class sha512_256 extends sha512 {
+/** @deprecated use @noble/hashes instead */
+export class sha512_256 extends shaJsSha512 {
```

Comment:
I'd be happy to remove and treat it as an internal (not triggering breaking change). Otherwise if we want to treat as external and used by consumers, I would keep this change to a minor/patch and remove the deprecated with the next major release.

But honestly, it's unlikely folks used this undocumented helper, even if it was `export`ed, so I'd also prefer treating this (and similar methods) as internal.